code/modules/clothing/clothing.dm

```diff
@@ -678,6 +678,10 @@ BLIND     // can't see anything
 	return ..()
 
 /obj/item/clothing/under/proc/can_attach_accessory(obj/item/clothing/accessory/A)
+	if(istype(A))
+		. = TRUE
+	else
+		return FALSE
```

Comment:
```suggestion
	if(!istype(A))
		return FALSE
```

Reply:
No need for an if else return, just return false if it's not what we want and let the proc continue normally otherwise

Reply:
I'm keeping this istype check removed and adding a proper check in quick equip